.github/workflows/cpplint_modified_files.yml

```diff
@@ -42,7 +42,8 @@ jobs:
           print("g++:")
           # compile_exts = tuple(".c .c++ .cc .cpp .cu .cxx".split())
           # compile_files = [file for file in cpp_files if file.lower().endswith(compile_exts)]
-          subprocess.run(["g++"] + cpp_files, check=True, text=True)
+          for cpp_file in cpp_files:
+            subprocess.run(["g++", cpp_file], check=True, text=True)
```

Comment:
Why is this deceleration of the script useful?

Reply:
This fixes the above error wherein g++ is compiling multiple files simultaneously

Reply:
Link to the error details: https://github.com/TheAlgorithms/C-Plus-Plus/pull/795#issuecomment-633790790

Reply:
Makes sense now.